### PR TITLE
Update chemical reactor recipes

### PR DESCRIPTION
**Pull request by Gurkysword12**

### CODE REVIEW

File: common/src/main/java/muramasa/gregtech/data/Materials.java [hunk 1]
```diff
@@ -583,8 +582,6 @@ public class Materials {
 
     public static Material Antimatter = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "anti_matter", 0x8000c4, NONE));
     public static Material BlueVitriol = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "blue_vitriol_water_solution", 0xffffff, NONE));
-
-    public static Material CalciumAcetateSolution = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "calcium_acetate_solution", 0xDCC8B4, NONE));
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/Materials.java]: somehow this got removed in the merge, it needs to be readded

File: common/src/main/java/muramasa/gregtech/data/Materials.java [hunk 2]
```diff
@@ -554,7 +554,6 @@ public class Materials {
     public static Material Ethanediol = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "ethanediol", 0xff8000, NONE));
     public static Material Ethanol = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "ethanol", 0xff8000, NONE));
     public static Material Ethenol = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "ethenol", 0xff8000, NONE));
-    public static Material EthylTertButylEther = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "ethyl_tert_butyl_ether", 0xffffff, NONE));
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/Materials.java]: Same as above

File: common/src/main/java/muramasa/gregtech/data/Materials.java [hunk 3]
```diff
@@ -620,19 +617,17 @@ public class Materials {
...
     public static Material Diesel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "diesel", 0xffff00, NONE));
     public static Material FishOil = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "fish_oil", 0xffc400, NONE));
-    public static Material HeavyFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "heavy_fuel", 0xffff00, NONE));
-    public static Material NitroFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "cetane_boosted_diesel", 0xc8ff00, NONE));
-    public static Material LightFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "light_fuel", 0xffff00, NONE));
+    public static Material HeavyFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "heavy_diesel", 0xffff00, NONE));
+    public static Material CetaneBoostedDiesel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "cetane_boosted_diesel", 0xc8ff00, NONE));
+    public static Material LightFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "light_diesel", 0xffff00, NONE));
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/Materials.java]: the merge change the names of light and heavy fuel to light and heavy diesel, need to change it back

File: common/src/main/java/muramasa/gregtech/data/Materials.java [hunk 4]
```diff
@@ -620,19 +617,17 @@ public class Materials {
...
     public static Material RocketFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "rocket_fuel", 0xffffff, NONE));
-    public static Material Gasoline = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "gasoline", 0xFFA500, NONE));
-    public static Material HighOctaneGasoline = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "high_octane_gasoline", 0xFFA500, NONE));
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/Materials.java]: got removed in merge, need to add it back

File: common/src/main/java/muramasa/gregtech/data/Materials.java [hunk 5]
```diff
@@ -620,19 +617,17 @@ public class Materials {
...
     public static Material OilHeavy = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "heavy_oil", 0x0a0a0a, NONE));
     public static Material OilMedium = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "raw_oil", 0x0a0a0a, NONE));
     public static Material OilLight = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "light_oil", 0x0a0a0a, NONE));
     public static Material SeedOil = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "seed_oil", 0xc4ff00, NONE));
-    public static Material SulfuricLightFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "sulfuric_light_fuel", 0xffff00, NONE));
-    public static Material SulfuricHeavyFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "sulfuric_heavy_fuel", 0xffff00, NONE));
+    public static Material SulfuricLightFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "sulfuric_light_diesel", 0xffff00, NONE));
+    public static Material SulfuricHeavyFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "sulfuric_heavy_diesel", 0xffff00, NONE));
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/Materials.java]: need to change name to sulfuric light and heavy fuel

File: common/src/main/java/muramasa/gregtech/data/Materials.java [hunk 6]
```diff
@@ -674,7 +669,6 @@ public class Materials {
     public static Material DinitrogenTetroxide = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "dinitrogen_tetroxide", 0x004184, NONE));
     public static Material HydrogenSulfide = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydrogen_sulfide", 0xffffff, NONE));
     public static Material NitricOxide = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "nitric_oxide", 0x7dc8f0, NONE));
-    public static Material NitrousOxide = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "nitrous_oxide", 0x7DC8FF, NONE));
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/Materials.java]: need to add back

File: common/src/main/java/muramasa/gregtech/data/Materials.java [hunk 7]
```diff
@@ -686,23 +680,23 @@ public class Materials {
...
-    //public static Material HydroCrackedPropene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_propene", 0xffdd55, NONE));
+    public static Material HydroCrackedPropene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_propene", 0xffdd55, NONE));
     public static Material HydroCrackedButane = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_butane", 0xb6371e, NONE));
-    //public static Material HydroCrackedButene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_butene", 0xcf5005, NONE));
-    //public static Material HydroCrackedButadiene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_butadiene", 0xe86900, NONE));
+    public static Material HydroCrackedButene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_butene", 0xcf5005, NONE));
+    public static Material HydroCrackedButadiene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_butadiene", 0xe86900, NONE));
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/Materials.java]: the merge uncommented them, need to be recommented

File: common/src/main/java/muramasa/gregtech/data/Materials.java [hunk 8]
```diff
@@ -686,23 +680,23 @@ public class Materials {
...
-    //public static Material SteamCrackedPropene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_propene", 0xffdd55, NONE));
+    public static Material SteamCrackedPropene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_propene", 0xffdd55, NONE));
     public static Material SteamCrackedButane = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butane", 0xb6371e, NONE));
-    //public static Material SteamCrackedButene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butene", 0xcf5005, NONE));
-    //public static Material SteamCrackedButadiene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butadiene", 0xe86900, NONE));
+    public static Material SteamCrackedButene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butene", 0xcf5005, NONE));
+    public static Material SteamCrackedButadiene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butadiene", 0xe86900, NONE));
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/Materials.java]: merge uncommented them, need to be recommented

**Review by Trinsdar — CHANGES_REQUESTED** [review]

File: common/src/main/java/muramasa/gregtech/data/Materials.java (hunk 6 above)
Comment (Gurkysword12) [reply common/src/main/java/muramasa/gregtech/data/Materials.java]: Should already be added

File: common/src/main/java/muramasa/gregtech/data/Materials.java (hunk 8 above)
Comment (Gurkysword12) [reply common/src/main/java/muramasa/gregtech/data/Materials.java]: Fixed

**Review by Trinsdar — APPROVED** [review]
looks good

**Pull request merged**